compiler-core/src/format/tests.rs

```diff
@@ -5377,3 +5377,32 @@ fn function_call_close_to_line_limit() {
 "#
     );
 }
+
+#[test]
+fn multiline_string_are_not_broken_with_string_concatenation_if_they_fit() {
+    assert_format!(
+        r#"pub fn main() {
+  "pub fn foo(" <> arg <> ") ->" <> type_ <> "{
+    body
+}"
+}
+"#
+    );
+}
+
+#[test]
+fn multiline_string_get_broken_on_newlines_as_function_arguments() {
+    assert_format!(
+        r#"pub fn main() {
+  foo(
+    bar,
+    "bar
+  asd
+       baz",
+    foo,
+    bar,
+  )
+}
+"#
+    );
```

Comment:
Could we have a test for an expression to the right of a multi-line string? To ensure that indentation works after it being set.

```gleam
pub fn main() {
  {
    let x = {
      "
1
2
"
      <> long_name_function_call(
        1111111111111111,
        222222222222,
        3333333333333333,
      )
  }
}
```
Or whatever the correct format is. The important bit is that the level of indentation afterwards is correct.

Reply:
Added!